sdcm/tester.py

```diff
@@ -502,6 +503,20 @@ def argus_collect_packages(self):
         except Exception:  # pylint: disable=broad-except
             self.log.error("Unable to collect package versions for Argus - skipping...", exc_info=True)
 
+    def argus_collect_manager_version(self):
+        if self.monitors:
```

Comment:
you might want to limit it, if we enable manager or not, we have a flag for that in configuration

Reply:
Do you mean "n_monitor_nodes"?

Reply:
`use_mgmt` param